cucumber/authenticators_gcp/features/step_definitions/authn_gcp_steps.rb

```diff
@@ -0,0 +1,23 @@
+
+Given(/^I set a GCP annotation with name: "([^"]*)" and value: "([^"]*)" to host: "([^"]*)"$/) do |hostname, annotation_name, annotation_value|
+    i_have_a_resource "host", hostname
+
+    if AuthenticatorHelpers.method_defined? annotation_value
+      set_annotation_to_resource(annotation_name,
+                                 Class.new.extend(AuthenticatorHelpers).send(annotation_value.to_s))
+    else
+      set_annotation_to_resource(annotation_name, annotation_value)
+    end
+end
+
+Given(/^I obtain a GCE identity token in "([^"]*)" format and audience claim value: "([^"]*)"$/) do | audience, format |
+    gce_identity_access_token(audience: audience, token_format: format)
+end
+
+Given(/I authenticate with gcp-authn using GCP identity token as host "([^"]*)"/) do |username|
+    authenticate_gcp_token(
+      account:     'cucumber',
+      username:    username,
```

Comment:
if you're using a host it should be `host/username`.

Reply:
only now we've added host